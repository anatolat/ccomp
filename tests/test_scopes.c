
int main() {
	int i = 11;
	{
		int i = 13;
		printf("%d\n", i);
	}
	printf("%d\n", i);
	{
		int i = 13;
		printf("%d\n", i);
	}
	printf("%d\n", i);

	for(int i = 15; 0;) printf("%d\n", i);

	printf("%d\n", i);
}