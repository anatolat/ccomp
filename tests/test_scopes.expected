13
11
13
11
11
